games-server/steamcmd/files/steamcmd.initd

```diff
@@ -0,0 +1,35 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+if [ "${SVCNAME}" = "steamcmd" ]; then
+	instance="main"
+else
+	instance="${SVCNAME#steamcmd.}"
+fi
+
+name="SteamCMD (Mod: ${instance})"
+pidfile="/run/steamcmd.${instance}.pid"
+screen_name="steamcmd.${instance}"
+start_stop_daemon_args="--chdir ${STEAMCMD_PATH}"
+
+command="screen"
+command_args="-DmUS ${screen_name} ${STEAMCMD_PATH}/${STEAMCMD_BINARY} ${STEAMCMD_OPTS}"
+command_background="true"
+command_group="steamcmd"
+command_user="steamcmd"
+
+depend() {
+	use net
+}
+
+start_pre() {
+	local OPTS
+	for OPTS in STEAMCMD_BINARY STEAMCMD_PATH STEAMCMD_OPTS; do
+	if [[ -z ${OPTS} ]]; then
```

Comment:
1. Bash is not fine in init.d. You need to use POSIX-ish shell.
2. This will never be true because OPTS is `STEAMCMD_BINARY`, etc.

Reply:
Ok, I've rewritten that part, that should be now POSIX?